core/src/main/resources/hudson/Messages.properties

```
@@ -79,14 +79,14 @@ ProxyConfiguration.Success=Success
 
 Functions.NoExceptionDetails=No Exception details
 
-PluginWrapper.missing={0} version {1} is missing. To fix, install version {1} or later.
-PluginWrapper.failed_to_load_plugin={0} version {1} failed to load.
-PluginWrapper.failed_to_load_dependency={0} version {1} failed to load. Fix this plugin first.
-PluginWrapper.disabledAndObsolete={0} version {1} is disabled and older than required. To fix, install version {2} or later and enable it.
-PluginWrapper.disabled={0} is disabled. To fix, enable it.
-PluginWrapper.obsolete={0} version {1} is older than required. To fix, install version {2} or later.
-PluginWrapper.obsoleteCore=You must update Jenkins from version {0} to version {1} or later to run this plugin.
-PluginWrapper.obsoleteJava=You must update Java from version {0} to version {1} or later to run this plugin.
+PluginWrapper.missing=Missing: {0} ({1})
+PluginWrapper.failed_to_load_plugin=Failed to load: {0} ({1})
+PluginWrapper.failed_to_load_dependency=Failed to load: {0} ({1})
+PluginWrapper.disabledAndObsolete=Update required: {0} ({1}) to {2} or higher
+PluginWrapper.disabled=Disabled: {0}
```

The message (also in the screen shot) does not state that this is a dependency. It seems unclear what this means because of that. It wasn't great before, but the "to fix, enable it" at least implied it.

Perhaps be a bit more verbose, keeping the new overall structure to allow for easier scanning?

```suggestion
PluginWrapper.disabled=Required plugin is disabled: {0}
```

(+ similar changes to other entries)